articles/connections/passing-parameters-to-idps.md

```diff
@@ -0,0 +1,127 @@
+  "strategy": "wordpress",
+  "options": {
+    "upstream_params": {
+      "blog": {
+        "alias": "access_type",
+        "value": "myblog.wordpress.com"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{ 
+      \"options\": { 
+        \"client_id\": \"...\", 
+        \"client_secret\": \"...\", 
+        \"upstream_params\": { 
+          \"blog\": { 
+            \"alias\": \"access_type\",
+            \"value\": \"myblog.wordpress.com\"
+          }
+        }, 
+        \"authorizationURL\": \"https://public-api.wordpress.com/oauth2/authorize\", 
+        \"tokenURL\": \"https://public-api.wordpress.com/oauth2/token\",
+        \"scope\": \"auth\" 
+      } 
+    }"
+  }
+}
+```
+
+You can test out your code and see other available attributes in our [Management API Explorer](https://auth0.com/docs/api/management/v2#!/Connections/post_connections).
+
+
+## Passing Dynamic Parameters
+
+Sometimes you may want to configure parameters per user. To do this, use the `upstream_params` element of the `options` attribute to specify a mapping between one of the existing accepted parameters to the parameter accepted by the Identity Provider. Then, when authorizing the user, dynamically add parameters to the authorization query.
+
+As an example, let's use Twitter, which allows you to pass an optional `screen_name` parameter to its OAuth authorization endpoint (for more information, see [Twitter's API reference](https://developer.twitter.com/en/docs/basics/authentication/api-reference/authorize)). Twitter's `screen_name` parameter pre-fills the username input box of the login screen with the given value, so we want to map it to the existing accepted parameter of `login_hint`.
+
+```
+{
+  "name": "TwitterConn",
+  "strategy": "twitter",
+  "options": {
+    "upstream_params": {
+      "screen_name": {
+        "alias": "login_hint"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{ 
+      \"options\": { 
+        \"client_id\": \"...\", 
+        \"client_secret\": \"...\", 
+        \"upstream_params\": { 
+          \"screen_name\": { 
+            \"alias\": \"login_hint\" 
+          } 
+        },  
+        \"authorizationURL\": \"https://api.twitter.com/oauth/authorize\",
+        \"tokenURL\": \"https://api.twitter.com/oauth/request_token\" 
+      } 
+    }"
+  }
+}
+```
+
+Now, when calling the /authorize endpoint for a specific user, you can pass their email address in the `login_hint` parameter:
+
+```text
+https://customer0.myauth0.com/authorize
```

Comment:
instead of dummy tenant names we use the variable `${account.namespace}`
so this `https://customer0.myauth0.com/authorize` should be `https://${account.namespace}/authorize`

see https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables